electrum_nmc/electrum/gui/qt/configure_dns_dialog.py

```diff
@@ -135,6 +135,16 @@ def __init__(self, value, parent):
         self.ui.btnEditRecord.clicked.connect(self.edit_selected_record)
 
         self.accepted.connect(lambda: self.name_dialog.set_value(self.get_value()))
+        
+        self.ui.editAHostname.textEdited.connect(self.validate_address_input)
```

Comment:
We should also revalidate when the address type drop-down is changed (e.g. from IPv4 to Tor).